pkg/machine/qemu/machine_windows.go

```diff
@@ -0,0 +1,38 @@
+package qemu
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+func isProcessAlive(pid int) bool {
+	dnr, err := os.OpenFile(os.DevNull, os.O_RDONLY, 0755)
+	if err != nil {
+		return err
+	}
+	defer dnr.Close()
+	dnw, err := os.OpenFile(os.DevNull, os.O_WRONLY, 0755)
+	if err != nil {
+		return err
+	}
+	defer dnw.Close()
+	files := []*os.File{dnr, dnw, dnw}
+	cmd := exec.Command("cmd", "/C", fmt.Sprintf("tasklist /FO csv /NH | findstr /L ,\"%d\",", pid))
+	cmd.Stdin = dnr
+	cmd.Stdout = dnw
+	cmd.Stderr = dnw
+	cmd.ExtraFiles = files
+	err := cmd.Run()
+	if err == nil {
+		return true
+	}
+	return false
+}
+
+func checkProcessStatus(processHint string, pid int, stderrBuf *bytes.Buffer) error {
+	if (isProcessAlive(pid) == false) {
```

Comment:
If you agree to switch the approach like in the previous comment, I would flip these so that isProcessAlive, just calls the state check method and verifies the return.

Reply:
Change applied and rebased to latest main.

Reply:
Converted to draft. Need some fixes (export function). Still learning how Go works, because it is my first experience with the language.

Reply:
Compilation issues fixed.